policy/centos7/k3s.fc

```diff
@@ -1,20 +1,21 @@
 # vim: sw=8:ts=8:et
-/etc/systemd/system/k3s.*						--	gen_context(system_u:object_r:container_unit_file_t,s0)
-/usr/bin/k3s								--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
-/usr/lib/systemd/system/k3s.*						--	gen_context(system_u:object_r:container_unit_file_t,s0)
-/usr/local/bin/k3s							--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
-/var/lib/cni(/.*)?								gen_context(system_u:object_r:container_var_lib_t,s0)
-/var/lib/rancher/k3s(/.*)?							gen_context(system_u:object_r:container_var_lib_t,s0)
-/var/lib/rancher/k3s/data(/.*)?							gen_context(system_u:object_r:container_runtime_exec_t,s0)
-/var/lib/rancher/k3s/storage(/.*)?						gen_context(system_u:object_r:container_file_t,s0)
-#/var/lib/rancher/k3s/agent/containerd/[^/]*/snapshots(/.*)?			gen_context(system_u:object_r:container_share_t,s0)
-/var/lib/rancher/k3s/agent/containerd/[^/]*/snapshots			-d	gen_context(system_u:object_r:container_share_t,s0)
-/var/lib/rancher/k3s/agent/containerd/[^/]*/snapshots/[^/]*		-d	gen_context(system_u:object_r:container_share_t,s0)
-/var/lib/rancher/k3s/agent/containerd/[^/]*/snapshots/[^/]*/.*			<<none>>
-/var/lib/rancher/k3s/agent/containerd/[^/]*/sandboxes(/.*)?			gen_context(system_u:object_r:container_share_t,s0)
-/var/run/flannel(/.*)?								gen_context(system_u:object_r:container_var_run_t,s0)
-/var/run/k3s(/.*)?								gen_context(system_u:object_r:container_var_run_t,s0)
-/var/run/k3s/containerd/[^/]*/sandboxes/[^/]*/shm(/.*)?				gen_context(system_u:object_r:container_runtime_tmpfs_t,s0)
-/var/lib/kubelet/pods(/.*)?							gen_context(system_u:object_r:container_file_t,s0)
-/var/log/containers(/.*)?							gen_context(system_u:object_r:container_log_t,s0)
-/var/log/pods(/.*)?								gen_context(system_u:object_r:container_log_t,s0)
+
+/etc/systemd/system/k3s.*                                       --  gen_context(system_u:object_r:container_unit_file_t,s0)
+/usr/lib/systemd/system/k3s.*                                   --  gen_context(system_u:object_r:container_unit_file_t,s0)
+/usr/local/lib/systemd/system/k3s.*                             --  gen_context(system_u:object_r:container_unit_file_t,s0)
+/usr/local/s?bin/k3s                                            --  gen_context(system_u:object_r:container_runtime_exec_t,s0)
+/usr/s?bin/k3s                                                  --  gen_context(system_u:object_r:container_runtime_exec_t,s0)
+/var/lib/cni(/.*)?                                                  gen_context(system_u:object_r:container_var_lib_t,s0)
+/var/lib/kubelet/pods(/.*)?                                         gen_context(system_u:object_r:container_file_t,s0)
```

Comment:
To be removed once containers/container-selinux#151 lands in centos/rhel/fedora-land.

Reply:
ah, nm, cent 7